providers/match.rb

```diff
@@ -0,0 +1,75 @@
+#
+# Cookbok Name:: td-agent
+# Provider:: match
+#
+# Author:: Pavel Yudin <pyudin@parallels.com>
+#
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+use_inline_resources
```

Comment:
Please use `use_inline_resources` only if it is defined, like `apt` cookbook is doing https://github.com/opscode-cookbooks/apt/blob/86c78b1867bc3861d652467f33addbdc521078bb/providers/repository.rb#L20